src/wp-includes/media.php

```diff
@@ -1903,6 +1910,44 @@ function wp_filter_content_tags( $content, $context = null ) {
 	return $content;
 }
 
+/**
+ * Adds a fallback mechanism to replace webp images with jpeg alternatives on older browsers.
+ *
+ * @since 6.1.0
+ */
+function wp_wepb_fallback() {
```

Comment:
See other comments, let's name this something like `wp_print_image_mime_fallback_script()`.